c2corg_ui/static/affiz.html

```diff
@@ -0,0 +1,14 @@
+<html>
+<head>
+<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
+<meta http-equiv="content-language" content="fr-FR" />
+<meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7" />
+<title></title>
+</head>
+<body leftmargin='0' topmargin='0' marginwidth='0' marginheight='0' style="background-color:transparent;">
+<div class="ads_323739345f343337315f3232393832"><script type="text/javascript">
+var rdads=new String(Math.random()).substring (2, 11);
+document.write('<sc'+'ript type="text/javascript" src="//server1.affiz.net/tracking/ads_display.php?n=323739345f343337315f3232393832_38888818f6&rdads='+rdads+'"></sc'+'ript>');
```

Comment:
See http://stackoverflow.com/questions/236073/why-split-the-script-tag-when-writing-it-with-document-write